core/src/nodes/handled_node.rs

```diff
@@ -160,6 +159,22 @@ where
     is_shutting_down: bool
 }
 
+impl<TMuxer, THandler> fmt::Debug for HandledNode<TMuxer, THandler>
+where
+    TMuxer: StreamMuxer,
+    THandler: NodeHandler<Substream = Substream<TMuxer>>,
+    THandler: fmt::Debug,
+{
+    fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
```

Comment:
How about using `fmt::Result`?

Reply:
Let's use the new suggestion API?

```suggestion
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result<()> {
```